src/libos/src/process/do_futex.rs

```diff
@@ -293,13 +298,13 @@ impl FutexBucket {
     }
 
     // TODO: this is an O(N) operation. Try to make it more efficient
-    /*pub fn dequeue_item(&mut self, futex_item: &FutexItem) -> Option<FutexItem> {
+    pub fn dequeue_item(&mut self, futex_item: &FutexItem) -> Option<FutexItem> {
```

Comment:
I am confused. If `dequeue_item` is disabled before, how could the original code compile?

Reply:
it's because that the code using it is commented on before.
```
/*
        let (_, futex_bucket_ref) = FUTEX_BUCKETS.get_bucket(futex_item.key);
        let mut futex_bucket = futex_bucket_ref.lock().unwrap();
        futex_bucket.dequeue_item(&futex_item);
*/
```